core/src/main/java/org/jdbi/v3/core/mapper/reflect/internal/PojoTypes.java

```diff
@@ -14,7 +14,7 @@
 package org.jdbi.v3.core.mapper.reflect.internal;
 
 import java.lang.reflect.Type;
-import java.util.HashMap;
+import java.util.concurrent.ConcurrentHashMap;
```

Comment:
order is wrong, thus fails the build. please run the build locally (`make install` in the root folder).